queue_job/jobrunner/runner.py

```diff
@@ -179,21 +183,38 @@ def _odoo_now():
     return _datetime_to_epoch(dt)
 
 
+def _connection_info_for(db_name):
+    db_or_uri, connection_info = odoo.sql_db.connection_info_for(db_name)
+
+    for p in ('host', 'port'):
```

Comment:
`s/p/[key|param]`

Reply:
If you don't mind, I will keep it as it to keep it consistent with the same piece of code that is in the other versions of the module.

Reply:
yeah, sorry, I missed the original code :sweat_smile:  :+1: